src/main/java/Counter.java

```diff
@@ -0,0 +1,14 @@
+import org.apache.log4j.Logger;
+
+public class Counter {
+    private static final Logger logger = Logger.getLogger(Counter.class);
+    private static final int MIN = 0;
+    private static final int MAX = 100;
+    private int counter = MIN;
+
+    public void counter(String threadName) {
```

Comment:
```suggestion
    public void count(String threadName) {
```
method names are usually verbs